pkg/wasm/convert.go

```diff
@@ -61,7 +92,16 @@ func MaybeConvertWasmExtensionConfig(resources []*anypb.Any, cache Cache) bool {
 
 func convert(resource *anypb.Any, cache Cache) (newExtensionConfig *anypb.Any, sendNack bool) {
 	ec := &core.TypedExtensionConfig{}
-	newExtensionConfig = resource
+	denyAll := false
+	defer func() {
+		if newExtensionConfig == nil {
+			var err error
+			newExtensionConfig, err = createFallbackFilter(ec.GetName(), denyAll)
```

Comment:
ECDS require type_url to allow-list extension types, RBAC in this case.

Reply:
Ah... will check it out. 
Is there any pointer?

Reply:
https://www.envoyproxy.io/docs/envoy/latest/api-v3/config/core/v3/config_source.proto#config-core-v3-extensionconfigsource field type_urls.